d2bs/kolbot/tools/Party.js

```diff
@@ -12,6 +12,7 @@ function main() {
 	include("common/Runewords.js");
 	include("common/Misc.js");
 	include("common/Prototypes.js");
+	include("common/Town.js");
```

Comment:
add newline after this
was this include removed previously?

Reply:
There was no include previously. This is now needed because of NPC variable change recently. Character config file didn't recognize these variables because they are defined in town.js file.